idea/common/src/interfaces/meta.ts

```diff
@@ -1,7 +1,7 @@
 import { ProgramMetadata } from '@gear-js/api';
 
 export interface IMeta {
-  id: number;
+  id: string;
```

Comment:
Let's name it `hash`

Reply:
I don't think so, it's not a good idea, because this `interface` have used a `meta` entity.

Reply:
Done